README.rst

```diff
@@ -69,3 +69,49 @@ Tests for code without Binary Ninja interaction can be run if you have a headles
 Adding a new dependency can be done by editing ``requirements.txt`` or ``requirements-dev.txt`` and then running the following in the virtual environment::
 
     $ pip install -r requirements-dev.txt -r requirements.txt
+
```

Comment:
Nit:Overall this looks good to me but can we also note somewhere that EVM is unsupported/unstable. Maybe near the "❗ATTENTION❗"

Reply:
Good idea! Added a [note](https://github.com/trailofbits/MUI/pull/24/commits/84e2f161b3759e9c185c24c0da0d7a752d8657ba#diff-7b3ed02bc73dc06b7db906cf97aa91dec2b2eb21f2d92bc5caa761df5bbc168fR12) to emphasize the lack of EVM support at the moment